polylogue/config.py

```
@@ -138,6 +139,19 @@ def persist_config(
             },
         },
     }
+    if roots:
+        mapped: Dict[str, Dict[str, str]] = {}
```

Ruff (0.15.9)

[error] Undefined name `Dict`

(F821)